mmhuman3d/data/datasets/pipelines/transforms.py

```diff
@@ -209,6 +210,41 @@ def _flip_smpl_pose(pose):
     pose_flipped[2::3] = -pose_flipped[2::3]
     return pose_flipped
 
+def _flip_smplx_pose(pose):
+    """Flip SMPLX pose parameters horizontally.
+
+    Args:
+        pose (np.ndarray([63])): SMPLX pose parameters
+    Returns:
+        pose_flipped (np.ndarray([21,3]))
+    """
+    SIGN_FLIP = np.array([6, 7, 8, 3, 4, 5, 9, 10, 11, 15, 16, 17,
```

Comment:
Can we use [horizontal flip](https://github.com/open-mmlab/mmhuman3d/blob/561f271cff941a7ed4196fa4126bc747f1b4c1fb/mmhuman3d/data/datasets/pipelines/transforms.py#L292)?